student.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'person'
+
+# class for student
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name = 'unknown')
+    super(age, name)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly add a parent permission parameter to the initialize method as per requirement.

Code example
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```